plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Purchase.php

```diff
@@ -54,20 +54,52 @@ public function get_id() {
 	 * @return string
 	 */
 	public function get_title() {
-		$products = $this->get_paid_products_and_themes();
+		$products   = $this->get_paid_products_and_themes();
+		$first_product    = count( $products['remaining'] ) >= 1 ? $products['remaining'][0] : false;
+		$additional_count = count( $products['remaining'] ) - 1;
+
+		if ( $this->get_parent_option( 'use_completed_title' ) && $this->is_complete() ) {
+			return count( $products['remaining'] ) === 1
```

Comment:
I think this should be `purchaseable` right? As the `remaining` array will always be empty if this task is completed.
As currently I am getting this weird title:
<img width="737" alt="Screen Shot 2022-06-01 at 3 43 14 PM" src="https://user-images.githubusercontent.com/2240960/171482076-b8338cca-9297-4166-9eaf-96fc1e83b734.png">

Reply:
Same probably goes for $addtional_count and line 79. 
For additional_count probably need a `$completed_additional_count`.

Reply:
Ya, good point.  This doesn't work well as you complete purchases.  Updated.